src/module_2/module_2_EDA.md

```diff
@@ -0,0 +1,2197 @@
+```
+
+
+
+
+<div>
+<style scoped>
+    .dataframe tbody tr th:only-of-type {
+        vertical-align: middle;
+    }
+
+    .dataframe tbody tr th {
+        vertical-align: top;
+    }
+
+    .dataframe thead th {
+        text-align: right;
+    }
+</style>
+<table border="1" class="dataframe">
+  <thead>
+    <tr style="text-align: right;">
+      <th></th>
+      <th>number_of_orders</th>
+      <th>counts</th>
+      <th>percentage</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>0</th>
+      <td>1</td>
+      <td>3572</td>
+      <td>71.683725</td>
+    </tr>
+    <tr>
+      <th>1</th>
+      <td>2</td>
+      <td>698</td>
+      <td>14.007626</td>
+    </tr>
+    <tr>
+      <th>2</th>
+      <td>3</td>
+      <td>251</td>
+      <td>5.037126</td>
+    </tr>
+    <tr>
+      <th>3</th>
+      <td>4</td>
+      <td>148</td>
+      <td>2.970098</td>
+    </tr>
+    <tr>
+      <th>4</th>
+      <td>5</td>
+      <td>89</td>
+      <td>1.786073</td>
+    </tr>
+    <tr>
+      <th>5</th>
+      <td>6</td>
+      <td>58</td>
+      <td>1.163957</td>
+    </tr>
+    <tr>
+      <th>6</th>
+      <td>7</td>
+      <td>46</td>
+      <td>0.923139</td>
+    </tr>
+    <tr>
+      <th>7</th>
+      <td>8</td>
+      <td>32</td>
+      <td>0.642183</td>
+    </tr>
+    <tr>
+      <th>8</th>
+      <td>9</td>
+      <td>23</td>
+      <td>0.461569</td>
+    </tr>
+    <tr>
+      <th>9</th>
+      <td>10</td>
+      <td>15</td>
+      <td>0.301023</td>
+    </tr>
+    <tr>
+      <th>10</th>
+      <td>11</td>
+      <td>13</td>
+      <td>0.260887</td>
+    </tr>
+    <tr>
+      <th>11</th>
+      <td>12</td>
+      <td>7</td>
+      <td>0.140478</td>
+    </tr>
+    <tr>
+      <th>12</th>
+      <td>13</td>
+      <td>6</td>
+      <td>0.120409</td>
+    </tr>
+    <tr>
+      <th>13</th>
+      <td>14</td>
+      <td>10</td>
+      <td>0.200682</td>
+    </tr>
+    <tr>
+      <th>14</th>
+      <td>15</td>
+      <td>1</td>
+      <td>0.020068</td>
+    </tr>
+    <tr>
+      <th>15</th>
+      <td>16</td>
+      <td>2</td>
+      <td>0.040136</td>
+    </tr>
+    <tr>
+      <th>16</th>
+      <td>17</td>
+      <td>3</td>
+      <td>0.060205</td>
+    </tr>
+    <tr>
+      <th>17</th>
+      <td>18</td>
+      <td>1</td>
+      <td>0.020068</td>
+    </tr>
+    <tr>
+      <th>18</th>
+      <td>19</td>
+      <td>1</td>
+      <td>0.020068</td>
+    </tr>
+    <tr>
+      <th>19</th>
+      <td>20</td>
+      <td>1</td>
+      <td>0.020068</td>
+    </tr>
+    <tr>
+      <th>20</th>
+      <td>21</td>
+      <td>3</td>
+      <td>0.060205</td>
+    </tr>
+    <tr>
+      <th>21</th>
+      <td>22</td>
+      <td>2</td>
+      <td>0.040136</td>
+    </tr>
+    <tr>
+      <th>22</th>
+      <td>25</td>
+      <td>1</td>
+      <td>0.020068</td>
+    </tr>
+  </tbody>
+</table>
+</div>
+
+
+
+
+```python
+top20_df_users_merged_counts = df_users_merged_counts.sort_values(by='counts', ascending=False).head(20)
```

Comment:
I would cut the queue later since everything after 11 is close to 0. You can always put a new bar which represents the counts of >11 orders